### ISSUE TITLE
Miscellaneous questions and wishes

### ISSUE THREAD

**Issue by Marcello1970** [vanBassum/StuffDatabase#2]
First, thank you for your time and afford. 
I have used the software for quit a while, and I have some questions / wishes
- What does the button 'Fetch TOR' do?
- Is it possible to put the components in alphabetical order?
- Add an field box number, so you can find te component in the right box.
- Add an warning when you press delete.
- make use of '100pF' or '100µF' instead of the full value, also at the resistors
- Add an IC tab. I think there are to many different properties to add. So only type and function would be great.

**Comment by vanBassum** [comment]
No problem, 
This will only improve the application and make it more accessible for other users.

- The 'Fetch TOR' button doesn't do anything, It was a test to see if it's possible to parse transistor information from alltransistors.com. This however doesn't work reliably enough to release to the public.
- I've added alphabetical sorting, although this only becomes active after closing and opening the application again. New items are added at the bottom of the list. By reopening the application this becomes sorted again. I might change this behaviour in the future.
- I've added a field 'BoxID'
- I've added a message box when deleting items.
- The values should be entered in normal or E notation, when printing they get converted to use SI units automatically. Right now the application doesn't support these units in the propertygrid (List of fields). Might add this later to the application.
- I've added a tab 'GenericIC'. 

See https://github.com/vanBassum/StuffDatabase/releases/tag/V1.0.2 for a release with these features added. Also, I'm made some progress on making the application more versatile, by enabling users to add, delete and edit components. There are some last things that don't work as intended yet and need some attention.
